### ISSUE TITLE
Add distance-from regions + directional changes to ratio fitness alert

### ISSUE THREAD

**Issue by zepumph** [phetsims/ratio-and-proportion#152]
In addition to description about the position of the slider hand you are moving, we also want to add in content about the relative distance/direction-of-movement from/to the other hand.

User this table for the directional regions: https://docs.google.com/document/d/1HJvvGWYaUv9tlat9S7omQQVng8fGB4FuUhTEM1qx1z8/edit#heading=h.mxsosk6hxekb

We also noted two cases where direction changes relative to the position of the other hand:

1. opposite arrow key, i.e. using up arrow, then using down arrow
2. same arrow keys moving past the position of the other hand

We decided to ignore (2) for now, and just implement (1) first.

Some example sentences are:

{{Extremely close to}} right hand, {{at}} challenge ratio‬.
{{Closer to}} right hand, {{somewhat close to}} challenge ratio.

**Comment by zepumph** [comment]
[the above commit](https://github.com/phetsims/ratio-and-proportion/commit/87eaef7b3b6abc924018fb5f8e8509f99aa70669) was good cleanup that separated describer logic into two types, one for the ratio fitness description, and the other about the position of the hands. This will make it easier to add these distance-related ones now. Right now I think they will go well in `HandPositionsDescriber.js`.

**Comment by zepumph** [comment]
* I implemented the regions as described in the table (not yet committed), and I feel a bit like "extremelyCloseTo" has a bit too much breadth (range of 0<X<2. It is 1/5 of the vertical distance. That much range doesn't seem like an extreme to me.

**Comment by zepumph** [comment]
This was added above, and can be tested in https://phet-dev.colorado.edu/html/ratio-and-proportion/1.0.0-dev.56/phet/ratio-and-proportion_a11y_view.html.

To review:
* Note that you can never get a direction-change alert when both hands' position is the same ("even with" distance).
* I'm not totally convinced that the region ranges are totally spot on. But I didn't notice until I could play with it. @terracoda see https://github.com/phetsims/ratio-and-proportion/blob/5f807bcc008db0e0861b306726cb20f33d7bc8da/js/common/view/HandPositionsDescriber.js#L154-L186 for the code version of the region ranges, and let me know if you want tweaking.

**Comment by zepumph** [comment]
We discussed this today, and thought that @Matthew-Moore240 should look at the above regions ranges and recommend any tweaks needed. He agreed that "extremely close to" had too large of a range.

**Comment by Matthew-Moore240** [comment]
Okay I think the new regions should be something like this 

This is with the right hand all the way to the top and the left hand all the way at the bottom.

**Left hand:**

- extremely far from (1.0 -.8) this is tick marks from 0 to 2 
- very far from(.7999 to .6) this is tick marks 2.1 to 4
- far from(.59999 to .4) this is tick mark 4.1 to 6
- not so close to (.3999 to .3) this is tick mark 6.1 to 7
- somewhat close to (.29999 to .2) this is tick mark 7.1 to 8 
- very close to (.19999 to .05) 8.1 to 9.5
- extremely close to (.04999 to .000001) this 9.6 to 9.9
- even with (0) when both hands are even

I hope this is somewhat clear @zepumph if you have any questions let me know

**Comment by zepumph** [comment]
Implemented above. This feels really good to me. One thing: 1.0 is currently a single point region "farthest from", is that still correct to you?

Please review. Anything else?

**Comment by Matthew-Moore240** [comment]
Yes thats totally good with me!
